controllers/usersTalentsController.js

```diff
@@ -1,30 +1,24 @@
 var db = require('../models');
 
-// GET '/api/users/:userId/talents'
-function index(req, res) {
-}
-
 // POST '/api/users/:user_id/talents'
 function create(req, res) {
   db.User.findById(req.params.user_id, function(err, foundUser) {
-    // foundUser = {name: 'Bob', talents: []};
     foundUser.talents.push(req.body);
     foundUser.save(function(err, savedUser) {
       res.json(savedUser);
     })
   })
 }
 
-// PUT '/api/users/:user_id/talents/:talentId'
```

Comment:
Your route in `server.js` is actually `/api/users/:user_id/talents/:talent_id` (throughout this file).